Web/starter_project_vanguard/components/layout/navbar.tsx

```diff
@@ -0,0 +1,51 @@
+import clsx from 'clsx'
+import Image from 'next/image'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+
+import logo from '../../public/images/a2sv-logo.png'
+import { NavItem } from '@/types/nav'
+
+const navItems: NavItem[] = [
+  { name: 'Home', href: '/' },
+  { name: 'Teams', href: '/team' },
+  { name: 'Success Stories', href: '/story' },
+  { name: 'About Us', href: '/about' },
+  { name: 'Get Involved', href: '/get-involved' },
+]
+
+const Navbar = () => {
+  const router = useRouter()
+  const currSegment = router.pathname.split('/')[1] || ''
+
+  return (
+    <div className="sticky top-0 bg-white bg-opacity-100 flex justify-between font-montserrat">
+      <Link href="/">
+        <Image src={logo} alt="A2SV Logo" />
+      </Link>
+      <div className="flex gap-4">
```

Comment:
this seems fine but can you check if tailwind has a tab that you can use for this list of links?
I think the animations and look of it might end up better if it exists

Reply:
![Screenshot 2023-04-22 3 33 06 PM](https://user-images.githubusercontent.com/88204792/233785255-7edd3d66-a970-41d7-8c7b-a53f47e99553.png)
when active the tab's color and border change, if that is what you are asking.

Reply:
yeah that is what I mean but I can guarantee you they probably implemented it better
It is fine to merge with this for now but I recommend you switch to an existing tab even if it is in another PR

https://tailwindui.com/components/application-ui/navigation/tabs
<img width="1024" alt="Screenshot 2023-04-22 at 10 30 34 PM" src="https://user-images.githubusercontent.com/39292639/233803101-282c81b2-3da4-402c-8400-c1fb2afdada7.png">
example from their website (which one do you think looks better?)

I recommend you make a habit of using things that are already built for you if it exists